test/commands/data/import/resume.nut.ts

```diff
@@ -36,18 +35,19 @@ describe('data import resume NUTs', () => {
 
     // about the type assertion at the end:
     // I'm passing `--json` in and `ensureExitCode: 0` so I should always have a JSON result.
-    const exportAsyncResult = execCmd<DataImportBulkResult>(
+    const importAsyncRes = execCmd<DataImportBulkResult>(
       `data import bulk --file ${csvFile} --sobject account --async --json`,
       { ensureExitCode: 0 }
     ).jsonOutput?.result as DataImportBulkResult;
 
-    expect(exportAsyncResult.jobId).not.to.be.undefined;
-    expect(exportAsyncResult.jobId).to.be.length(18);
+    expect(importAsyncRes.jobId).not.to.be.undefined;
+    expect(importAsyncRes.jobId).to.be.length(18);
 
-    const importResumeResult = execCmd<DataImportBulkResult>(
-      `data import resume -i ${exportAsyncResult.jobId} --json`,
-      { ensureExitCode: 0 }
-    ).jsonOutput?.result as DataImportBulkResult;
+    await validateCacheFile(path.join(session.homeDir, '.sf', 'bulk-data-import-cache.json'), importAsyncRes.jobId);
```

Comment:
since all bulk ingest commands share most of the logic we want to ensure the right cache file name is being used for each command